test/metabase/api/dashboard_test.clj

```diff
@@ -178,14 +178,16 @@
 ;;; +----------------------------------------------------------------------------------------------------------------+
 
 (deftest create-dashboard-validation-test
-  (testing "POST /api/dashboard"
-    (is (= {:errors {:name "value must be a non-blank string."}}
-           (mt/user-http-request :rasta :post 400 "dashboard" {})))
+  (mt/with-fake-events-collector [{:keys [events]}]
+    (testing "POST /api/dashboard"
+      (is (= {:errors {:name "value must be a non-blank string."}}
+             (mt/user-http-request :rasta :post 400 "dashboard" {})))
 
-    (is (= {:errors {:parameters (str "value may be nil, or if non-nil, value must be an array. "
-                                      "Each parameter must be a map with :id and :type keys")}}
-           (mt/user-http-request :crowberto :post 400 "dashboard" {:name       "Test"
-                                                                   :parameters "abc"})))))
+      (is (= {:errors {:parameters (str "value may be nil, or if non-nil, value must be an array. "
+                                        "Each parameter must be a map with :id and :type keys")}}
+             (mt/user-http-request :crowberto :post 400 "dashboard" {:name       "Test"
+                                                                     :parameters "abc"})))
+      (is (= 0 (count (events)))))))
```

Comment:
Same as
```suggestion
      (is (empty? (events))))))
```
but this has a marginally better error description

Reply:
yep 88da822e5d